frontend/src/scss/components/card.scss

```diff
@@ -1,5 +1,6 @@
 @mixin card($border-radius, $padding: 0) {
   @include u-padding($padding);
+
```

Comment:
Ideally, we can suppress adding a space after the include when the include is in the beginning of the declaration. Will need to look into this.

Reply:
Fixed in 6b1bf81